### ISSUE TITLE
c->cancelWithError(title, html);

### ISSUE THREAD

**Issue by pengguanjun** [Esri/dynamic-situational-awareness-qt#359]
I assume this is a problem.

AuthenticationController.cpp 296    c->cancelWithError(title, html);

C:\arcgis\1\dynamic-situational-awareness-qt\arcgis-runtime-toolkit-qt\uitools\cpp\Esri\ArcGISRuntime\Toolkit\AuthenticationController.cpp:296: 
error: no member named 'cancelWithError' in 'Esri::ArcGISRuntime::AuthenticationChallenge'

![图片](https://user-images.githubusercontent.com/29192658/116763536-52e86680-aa50-11eb-889a-ab8075f1f5bc.png)

**Comment by anmacdonald** [comment]
Good find. You are right, this is a problem with our build instructions.

We've just released ArcGISRuntime 100.1.1, but DSA is building against ArcGIS 100.10. 

The toolkit module you are using expects 100.11. 

There is an easy workaround here. When it coms to this instruction:

```bash
git clone https://github.com/Esri/arcgis-runtime-toolkit-qt.git
```

Run this instead

```bash
git clone --branch release/100.10 https://github.com/Esri/arcgis-runtime-toolkit-qt.git
```

I'll update the instructions to be more specific.

**Comment by pengguanjun** [comment]
Thx, handsome man.
It works.
git clone --branch release/100.10.0 https://github.com/Esri/arcgis-runtime-toolkit-qt

![图片](https://user-images.githubusercontent.com/29192658/117007668-7e9f7100-ad1c-11eb-8a79-d357274b61f7.png)
